sdk/keyvault/samples/keyvaultproxy/tests/AzureSamples.Security.KeyVault.Proxy.Tests.csproj

```diff
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
```

Comment:
This project is a sample meant to standalone on Samples Browser. That property won't exist normally. This change needs to be reverted.

Reply:
An alternative fix would be to initialize `RequiredTargetFrameworks` here if not already initialized. These need to work standalone when downloaded from Samples Browser. I spent way too much time on these (would've been easier in a separate repo; cc @weshaggard) to make sure they work in this repo as well as separately.